app/views/devise/registrations/edit.html.erb

```diff
@@ -39,13 +39,19 @@
   <div class="smallMarginBottom">
     <%= ProfilePicture.fetch(resource.avatar.url) %>
   </div>
-  <%= f.file_field :avatar, 'aria-label': t('devise.registrations.edit.labels.profile_picture') %>
+  <%= f.file_field :avatar, 'aria-label': t('devise.registrations.edit.labels.profile_picture'), style: 'margin-bottom: 5px' %>
   <% unless resource.avatar.blank? %>
     <%= f.label :remove_avatar, class: 'checkbox' do %>
       <%= f.check_box :remove_avatar, 'aria-label': t('devise.registrations.edit.labels.remove_picture') %>
       <%= t('devise.registrations.edit.labels.remove_picture') %>
     <% end %>
   <% end %>
+  <% if current_user.provider == 'google_oauth2' %>
```

Comment:
Hi @bmwachajr , yes the `no_third_party_avatar` field can be changed to true or false, defaulting with false, to enable the user to opt-out of having their third-party login provider avatar uploaded.

Reply:
What happens when a user opts in to have their third-party login provider avatar uploaded? When the third-party login provider is not `google_oauth2`.

Ther is a feature that a user can use to manually remove the avatar. cant a user just remove the third party avatar instead of opting in/out of `no_third_party_avatar`

Reply:
If the user logs in via `google_oauth2`, by default their Google Avatar will be uploaded as their profile image. If they select "remove avatar", it will remove their avatar for that log in session, but if they log in again via Google, the profile image will be uploaded again. 

We created the second field, `no_third_party_avatar` to enable the user to opt-out of the default third party avatar uploading. If they select the checkbox for `no_third_party_avatar`, the field will be "true" and remain true to ensure that their Google Avatar isn't uploaded each time they log in.  

Hopefully that makes sense, but let me know if you still have questions! 😃

Reply:
If the user logs in via `google_oauth2`, by default their Google Avatar will be uploaded as their profile image. If they select "remove avatar", it will remove their avatar for that log in session, but if they log in again via Google, the profile image will be uploaded again. 

We created the second field, `no_third_party_avatar` to enable the user to opt-out of the default third party avatar uploading. If they select the checkbox for `no_third_party_avatar`, the field will be "true" and remain true to ensure that their Google Avatar isn't uploaded each time they log in.  

Hopefully that makes sense, but let me know if you still have questions! 😃

Reply:
Yeah, that makes sense. thank you for clarifying 👍. On the flip side, we created a [cloudinary_service](https://github.com/ifmeorg/ifme/blob/master/app/services/cloudinary_service.rb) that we can use to upload avatars to cloudinary. What are your thoughts about using this service to upload avatars to couldinary?